stan/math/opencl/prim/col.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef STAN_MATH_OPENCL_PRIM_COL_HPP
+#define STAN_MATH_OPENCL_PRIM_COL_HPP
+#ifdef STAN_OPENCL
+
+#include <stan/math/opencl/matrix_cl.hpp>
+#include <stan/math/opencl/kernel_generator.hpp>
+
+namespace stan {
+namespace math {
+
+/** \ingroup opencl
```

Comment:
Should these be in a sub group like `methods`?

Reply:
I could go with a prim subgroup. These are all  just trying to add every missing prim signature in the Stan language. I dont think this will be used inside Math directly, much like the regular prim/col is not.